src/App.js

```diff
@@ -1,5 +1,146 @@
+const { Console, Random } = require('@woowacourse/mission-utils');
+const { MESSAGES, NUMBERS, UNITS, REGEX, CHOICE } = require('./constants');
+
 class App {
-  play() {}
+	computerNumArr = [];
+
```

Comment:
```suggestion
#restart = false
```
이것을 미리 설정해주면 어떨까요 ??